### DISCUSSION TITLE
GIT LFS with local repo on network drive

### DISCUSSION THREAD

**Discussion by GVAndyL** [git-lfs/git-lfs#4635] [Q&A] [answered]
I'm using git lfs 2.13.3 on Windows 10 and I have a local bare repo on a network drive. I am having problems pushing lfs files to the repo.

The remote repo address is a full UNC path of the form \\fs2\R&D\PP\testRepo.git and when I push to it I get

Uploading LFS objects: 100% (1/1), 219 KB | 0 B/s, done.
EOF
error: failed to push some refs to '\\fs2\R&D\PP\testRepo.git'

Turning on GIT_TRACE I see the following in the trace

17:37:09.081072 trace git-lfs: xfer[git-lfs]: error creating handler: chdir C:\Program Files\Git\fs2\R&D\PP\testRepo.git: The system cannot find the path specified.

Full trace is attached.

If I mount the remote drive with a drive letter so \\fs2\R&D is mounted as drive Z, then set the remote URL to be Z:\PP\testRepo.git everything works correctly. So something doesn't like the UNC path name.

I have also tried setting the remote URL to file://fs2/R&D/PP/testRepo.git, but that doesn't work either.

Is there a way to make this work without mounting the network drive?

Thanks for any help
[trace.txt](https://github.com/git-lfs/git-lfs/files/7201984/trace.txt)

**Comment by bk2204** [comment, marked as answer]
Hey,

You can try the syntax `file:////fs2/R%26D/PP/testRepo.git`, but right now we don't know whether that works.  The core team doesn't use Windows and doesn't have any such environment to test with, so we'd need someone to test this and possibly contribute a patch for it if that syntax doesn't work.

**Reply by GVAndyL** [reply]
Hi,

Thanks for the reply. That syntax looks like it works. I had tried something like that, but probably not enough slashes or the wrong sort.

Regards
